.env.example

```
@@ -0,0 +1,85 @@
+# ---------------------------------------------------------------------
+# Required for production: admin auth + API
+# ---------------------------------------------------------------------
+ADMIN_EMAIL=admin@example.com
+# ADMIN_PASSWORD_SECRET=arn:aws:secretsmanager:us-east-1:000000000000:secret:cudly-admin-password-PLACEHOLDER
+# API_KEY_SECRET_ARN=arn:aws:secretsmanager:us-east-1:000000000000:secret:cudly-api-key-PLACEHOLDER
+
+# ---------------------------------------------------------------------
+# Optional: web frontend / CORS / dashboard
+# ---------------------------------------------------------------------
+DASHBOARD_URL=http://localhost:3000
+CORS_ALLOWED_ORIGIN=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORS_ALLOWED_ORIGIN key should go before the DASHBOARD_URL key

(UnorderedKey)